### PR TITLE
Print information in gRPC CLI if its CQ event is not successful

### PR DESCRIPTION
**Pull request by y-zeng**

### CODE REVIEW

**Comment by grpc-testing** [comment]
```
****************************************************************

libgrpc.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]


****************************************************************

libgrpc++.so

     VM SIZE        FILE SIZE
 ++++++++++++++  ++++++++++++++

  [ = ]       0        0  [ = ]



```

**Comment by grpc-testing** [comment]
```
[trickle] No significant performance differences
```

**Comment by grpc-testing** [comment]
```
[microbenchmarks] No significant performance differences
```

**Comment by y-zeng** [comment]
CC @yang-g

File: test/cpp/util/cli_call.cc [hunk 1]
```diff
@@ -30,7 +30,11 @@
 namespace grpc {
 namespace testing {
 namespace {
-void* tag(int i) { return (void*)static_cast<intptr_t>(i); }
+void* const kTagStartCall = (void*)static_cast<intptr_t>(1);
```
Comment (dgquintas) [inline test/cpp/util/cli_call.cc]: you want to do `reinterpret_cast<void*>(1)`. Same for the rest of the lines.

**Comment by hcaseyal** [comment]
Closing because no response from the author in over a month.

**Pull request closed without merging**